test/lib/ufe/CMakeLists.txt

```diff
@@ -17,25 +17,22 @@ set(TEST_SUPPORT_FILES
 
 if(CMAKE_UFE_V2_FEATURES_AVAILABLE)
     list(APPEND TEST_SCRIPT_FILES
-        testGroupCmd.py
         testAttribute.py
         testAttributes.py
         testComboCmd.py
+        testContextOps.py
+        testDeleteCmd_v2.py
```

Comment:
I split out the UFE v2 delete test into a separate file to include here.

Reply:
You can see this block of tests are all V2 only. So most of the skipif tags have been removed. I had to keep a couple since they are v0.20.0.